pkgdown/_pkgdown.yml

```diff
@@ -63,6 +78,10 @@ navbar:
         href: articles/micropooling.html
       - text: Incorporating a Trajectory Model
         href: articles/trajectories.html
```

Comment:
We want to get rid of this because we are no longer supporting Trajectories on the front end.

Reply:
Don't we want to keep this since it still works in old versions?

Reply:
Sure, that's fine. Can you note at the top of this vignette that it is a deprecated feature as of the newest VISION version?

Reply:
*Note: this feature is deprecated in VISION 3.0+.*

Does that sound good?